vault/resource_aws_secret_backend_role_test.go

```diff
@@ -16,7 +16,7 @@ const testAccAWSSecretBackendRolePolicyArn_basic = "arn:aws:iam::123456789123:po
 const testAccAWSSecretBackendRolePolicyArn_updated = "arn:aws:iam::123456789123:policy/bar"
 
 func TestAccAWSSecretBackendRole_basic(t *testing.T) {
-	backend := acctest.RandomWithPrefix("tf-test-aws")
+	backend := acctest.RandomWithPrefix("tf-test-aws/nested")
```

Comment:
While this works I think you should just copy/paste it to `TestAccAWSSecretBackendRole_nested` or something so that we are also still testing the original path as well.